native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar.ios.js

```diff
@@ -134,7 +133,10 @@ class BpkNavigationBar extends Component<Props, {}> {
 
   constructor(props) {
     super(props);
-    this.theme = getThemeStyle(IOS_THEME_ATTRIBUTES, this.props.theme || {});
+    this.theme = getThemeAttributes(
+      IOS_THEME_ATTRIBUTES,
+      this.props.theme || {},
```

Comment:
Not a biggy, but it'd be nice if you could just pass `this.props.theme`.

Reply:
Suppose yeah